packages/chakra-ui/src/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Heading } from '@chakra-ui/react'
+import { List, ListIcon, ListItem } from '@chakra-ui/react'
+import { ErrorListProps } from '@rjsf/core'
+import React from 'react'
+
+const ErrorList = ({ errors }: ErrorListProps) => (
+  <Box mb={2} p={2}>
+    <Heading as="h6" size="md" fontWeight="500" color="red.600">Errors</Heading>
+    <List spacing={2}>
+      {errors.map((error, i) => (
+        <ListItem key={i}>
+          <ListIcon icon="warning-2" />
```

Comment:
I wonder if we can instead show red "X"'s instead of question marks? Let me know what icon you think looks best here.

![image](https://user-images.githubusercontent.com/1689183/111927295-99789780-8a86-11eb-823d-984c415fd71c.png)